.github/workflows/main.yaml

```diff
@@ -53,3 +60,25 @@ jobs:
           fail_ci_if_error: true
           files: ./coverage.xml
           flags: unittests
+  build-and-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+    needs:
+      - unit-tests
+      - integration-tests
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install pypa/build
+        run: python -m pip install build --user --upgrade
+      - name: Build a binary wheel and a source tarball
+        run: python setup.py sdist bdist_wheel
+      - name: Publish distribution 📦 to PyPI
```

Comment:
we can also store the .whl as artifact and make it downloadable from the UI. Then it is easier to let people test something. Here is written how it works:
https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts